angular/core/components/group_page/group_page_controller.coffee

```diff
@@ -62,11 +62,11 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $loca
     @group.noInvitationsSent() and
     !@group.trialIsOverdue() and
     !@subscriptionSuccess and
-    !GroupWelcomeModal.shownToGroup[@group.id]
+    !CurrentUser.experiences["seenWelcomeFor#{@group.key}"]
```

Comment:
I think this should be CurrentUser.hasExperienced('welcomeFor#{group.key}'). So that we abstract the implementation.